### PR TITLE
strip \r and \n when reading the file /etc/host_hostname

### PR DESCRIPTION
**Pull request by nvanheuverzwijn**
related to #329

### CODE REVIEW

**Comment by nvanheuverzwijn** [comment]
@michaelshobbs Will do. However, testing this might be difficult because of the hardcoded filename. I will make something up.

**Comment by nvanheuverzwijn** [comment]
@michaelshobbs Done. Waiting for your feedback.

File: adapters/syslog/syslog_test.go [hunk 1]
```diff
@@ -41,8 +43,46 @@ var (
...
+        hostHostnameContent = "hostname\r\n"
 )
 
+func TestHostnameDoesNotHaveLineFeed(t *testing.T) {
+	err := ioutil.WriteFile(hostHostnameFilename, []byte(hostHostnameContent), 0777)
+	if err != nil {
+		t.Fatal(err)
+	}
+	done := make(chan string)
+	addr, sock, srvWG := startServer("tcp", "", done)
```
Comment (michaelshobbs) [inline adapters/syslog/syslog_test.go]: looks like that broke the reconnect test. No need to spin up the server. I'd just move the hostname logic to it's own function and test that specifically.

Comment (nvanheuverzwijn) [reply adapters/syslog/syslog_test.go]: Hey, I'm sorry, I really don't have time to do this. Maybe someone with more experience in go should do this.

Comment (michaelshobbs) [reply adapters/syslog/syslog_test.go]: No worries. I'll take it from here. Thanks for the initial contribution!

**Comment by nvanheuverzwijn** [comment]
Duplicate #338

**Pull request closed without merging**